anoncreds/src/ffi/credential.rs

```diff
@@ -59,6 +59,7 @@ pub extern "C" fn anoncreds_create_credential(
     attr_names: FfiStrList,
     attr_raw_values: FfiStrList,
     attr_enc_values: FfiStrList,
+    rev_reg_id: FfiStr,
```

Comment:
Why wasn't this needed before?

Reply:
I believe it was inside the `rev_reg` object